README.md

```diff
@@ -0,0 +1,640 @@
+# endaq-ide
+
+`endaq.ide` contains high-level utility functions to aid in importing and inspecting enDAQ IDE recording files. Key functions/features include:
+
+* `get_channel_table()`: Get summary information about the contents of a file.
+* `get_doc()`: Open an IDE, either locally or from a URL.
+* `to_pandas()`: Convert IDE data to a `pandas` `DataFrame`, for use with other packages.
+
+*Note: This is a component of the `endaq` package; it is separate to allow users to install it independently, avoiding the core `endaq-python` dependencies. It is included in a full `endaq` install.*
+
+## `endaq.ide` usage examples
+Here are a few examples of `endaq.ide` in use. For more information, see the [full documentation]().
+
+For brevity, the following examples assume everything has been imported from `endaq.ide`:
+```python
+from endaq.ide import *
+```
+
+### Opening IDE files: `endaq.ide.get_doc()`
+`endaq.ide` includes a convenient shortcut for importing IDE data: `get_doc()`. It can load data from local files, or read data directly from a URL.
+
+
+```python
+doc = get_doc("tests/test.ide")
+doc1 = get_doc("https://mide.services/software/test.ide")
+```
+
+IDE files can be retrieved directly from Google Drive using a Drive 'sharable link' URL. The file must be set to allow access to "Anyone with the link."
+
+
+```python
+doc2 = get_doc("https://drive.google.com/file/d/1t3JqbZGhuZbIK9agH24YZIdVE26-NOF5/view?usp=sharing")
+```
+
+Whether opening a local file or a URL, `get_doc()` can be used to import only a specific interval by way of its `start` and `end` parameters:
+
+
+```python
+doc3 = get_doc("tests/test.ide", start="5s", end="10s")
+```
+
+### Summarizing IDE files: `endaq.ide.get_channel_table()`
+Once an IDE file has been loaded, `endaq.ide.get_channel_table()` will retrieve basic summary information about its contents.
+
+
+```python
+get_channel_table(doc)
+```
+
+
+
+
+<style type="text/css">
```

Comment:
These lines seem to be rendered as normal text, for some reason.

Reply:
Huh. Maybe Github disallows `<style>` tags because they could be used for malicious purposes.

Reply:
```html
<style> 
  p {font-family: hellvetica, sans-serif;} 
</style>
```